app/src/main/res/layout/list_item_book.xml

```diff
@@ -32,14 +42,14 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:textSize="@dimen/primary_text_size"
-            src:text="Sample text" />
```

Comment:
i tu tekst wpisany

Reply:
To chyba nie moje

Reply:
Możliwe! Ale wszyscy dbamy o kod. :) Najważniejsza zasada ze wszystkich: zawsze zostawiaj kod ładniejszy niż go zastałeś. (czy jakoś tak)
Ja i przyszłe pokolenia będziemy Ci wdzięczni :P

Reply:
Ok - będę o tym pamiętał 😉

Reply:
Mój błąd, powinno to być tools:text. Ale co ciekawe AS nie pokazuje warninga przy zahardcodowanym atrubucie src:text.

Reply:
Dlatego chyba nie zwraca błędu: 
xmlns:src="http://schemas.android.com/tools"